package.json

```diff
@@ -49,11 +49,11 @@
     "nodemon": "^1.11.0",
     "opn-cli": "^3.1.0",
     "react-addons-test-utils": "^15.4.1",
+    "react-hot-loader": "^3.0.0-beta.6",
```

Comment:
`react-hot-loader` needs to be a production dependency. The library itself handles `production`/`development` properly.